AnalyzerTools/src/GEScaleSyst.C

```diff
@@ -0,0 +1,2411 @@
+#ifndef HighPtMuonAnalysis_GEScale
+#define HighPtMuonAnalysis_GEScale
+
+#include "GEScaleSyst.h"
+
+using namespace std;
+
+ScaledPts GEScaleSyst::GEPt(int Year, float pt, float eta, float phi, int charge){
+
+  ScaledPts out;
+  out.ScaledPt = pt;
+  out.ScaeldPt_Up = pt;
+  out.ScaeldPt_Down = pt;
+
+  Year = Year-2000;
+
+  int i_nominal = Year * 100;
+  double Pt_Central = GEScaleCorrPt(i_nominal, pt, eta, phi, charge, false);
+
+  double sum = 0.;
+  for(int s=0; s<50; s++){
```

Comment:
maybe better to use the pre-increment here ++s? You use this is line 45.

Reply:
It's a copy of Min's repo and I don't wanna touch it

Reply:
Check https://github.com/jedori0228/GEScaleSyst/commit/189e026a8f736f437d26dba40630f0c2ebe1fc8b , I removed GEScaleSyst from AnalyzerTools/ and compile external/GEScaleSyst directly